src/instructlab/train/linux_train.py

```diff
@@ -289,31 +288,30 @@ def model_generate(user, **kwargs):
     )
 
     tokenizer.padding_side = "right"
-    output_dir = "./training_results"
-    per_device_train_batch_size = 1
+    train_args = Munch(
```

Comment:
Why did you introduce a new dependency? Wouldn't `dict()` work here, too?

Reply:
Munch is more readable than dict for accessing individual members.